Lesson34/Merve/Lesson33/props-drilling-app/src/modules/user/UserDetailsForm/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useContext } from "react";
+import { UserDispatchContext } from "../UserProvider";
+import './styles.css';
+
+export const UserDetailsForm = ({ username, onNameChange }) => {
```

Comment:
This component should not accept onNameChange as props. You get it from Context directly instead.